drivers/sensor/mhz19b/mhz19b-priv.h

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2021 G-Technologies Sdn. Bhd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_MHZ19B_MHZ19B_PRIV
+#define ZEPHYR_DRIVERS_SENSOR_MHZ19B_MHZ19B_PRIV
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(mhz19b, CONFIG_SENSOR_LOG_LEVEL);
```

Comment:
You are not using log subsys here so you can move it into the source file.

Reply:
I actually deliberately place that, alongside with all other required headers in the `mhz19b-priv.h` as it is a private header, is that not okay?

Reply:
I usually include the headers there where I use them but I don't know if there is a rule for it :)